SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,119 @@
+
+#### Using Locator
+The Locator is a SpatialOS service which allows you to connect to cloud deployments. 
+
+> Experimental: You can use `ClientTravel` to change which cloud deployment a client-worker is connected to. This functionality is highly experimental and requires you to write your own authorization code.
+
+Using the locator flow is very similar to using the receptionist, except with different URL options. You must add the `locator` option, and specify a project name (the SpatialOS Console project where you have started your deployment), a deployment name (the name you use to start the deployment), and a login token (requires writing authentication code, see [our authentication service docs](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication) for details). For example:
+
+```
+FURL TravelURL;
+TravelURL.Host = TEXT("locator.improbable.io");
+TravelURL.Map = TEXT("DesiredMap");
+TravelURL.AddOption(TEXT("locator"));
+TravelURL.AddOption(TEXT("project=MY_PROJECT_NAME"));
+TravelURL.AddOption(TEXT("deployment=MY_DEPLOYMENT_NAME"));
+TravelURL.AddOption(TEXT("token=MY_LOGIN_TOKEN"));
+
+PlayerController->ClientTravel(TravelURL.ToString(), TRAVEL_Absolute, false /*bSeamless*/);
+```
+
+### ClientTravel - technical details
+We have made changes to the Unreal Engine to detect if you have SpatialOS networking enabled. If you do, when you specify a ClientTravel URL containing a host to connect to, we create a `SpatialPendingNetGame` instead of a default Unreal `PendingNetGame`. This internally creates a `SpatialNetConnection` which connects you to the specified host. 
+
+## `UWorld::ServerTravel`
+> Warning: `ServerTravel` is in an experimental state and we currently only support it in single server-worker configurations.   
+> We don’t support `ServerTravel` in [PIE](https://docs.unrealengine.com/en-us/GettingStarted/HowTo/PIE#playineditor).
+
+### In native Unreal
+`ServerTravel` in Unreal is the concept of changing the [map (or Level - see Unreal documentation)](http://api.unrealengine.com/INT/Shared/Glossary/index.html#l) for the server and all connected clients. A common use case is starting a server in a lobby level. Clients connect to this lobby level and choose loadout, character etc. When ready, the server triggers a `ServerTravel`, which transitions the deployment and all clients into the main game level.
+
+When `ServerTravel` is triggered, the server tells all clients to begin to [`ClientTravel`](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling) to the map specified. If the `ServerTravel` is seamless then the client maintains its connection to the server. If it’s not seamless then all the clients disconnect from the server and reconnect once they have loaded the map. Internally, the server does a similar process: it loads in the new level, usually a game world for all the clients to play on, and begins accepting player spawn requests once ready.
+
+### User guide
```

Comment:
Again, can we say "In the GDK" so show it's the other side of "In native Unreal"?